ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -233,9 +201,41 @@ class ServicesHomeScreen extends React.Component<Props, State> {
       toastErrorMessage: EMPTY_MESSAGE,
       isInnerContentRendered: false
     };
+    this.updateLongPressButtonScope = this.updateLongPressButtonScope.bind(
+      this
+    );
+    this.handleOnLongPressItem = this.handleOnLongPressItem.bind(this);
+    this.onServiceSelect = this.onServiceSelect.bind(this);
+    this.showAlertOnDisableServices = this.showAlertOnDisableServices.bind(
```

Comment:
This is absolutely right!
Since this binding seems to generate too boilerplate I read this https://blog.g2i.co/react-class-components-with-es6-and-class-fields-927b2b59f94e

So we could revert to class field where is possibile.
What do you think?

Reply:
I have some doubts yet about it (even on the changes I made here). 
For example, after reading [this](https://medium.com/dailyjs/demystifying-memory-usage-using-es6-react-classes-d9d904bc4557) I opted to not use class fields, giving more importance to the memory saving: even if the author says he prefers to avoid the manual bind to the memory savings, it seems more about the style of the code, not the "efficiency" of it